torrent.go

```diff
@@ -710,6 +710,32 @@ func (t *Torrent) bitfield() (bf []bool) {
 	return
 }
 
+func (t *Torrent) CompletedPiecesBitfield() []bool {
```

Comment:
Can't you already do this by iterating over the Torrent.Pieces and getting their status individually?

Reply:
You can do that to check if a local piece progress but there is no exposed method or way to retrieve the remote status of a piece